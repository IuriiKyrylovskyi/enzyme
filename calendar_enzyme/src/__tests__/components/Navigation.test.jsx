import React from "react";
import { shallow } from "enzyme";
import Navigation from "../../components/navigation/Navigation";

const mockWeekDates = [new Date("2019-04-22T10:20:30Z"), new Date("2019-04-23T10:20:30Z"), new Date("2019-04-24T10:20:30Z"), new Date("2019-04-25T10:20:30Z"), new Date("2019-04-26T10:20:30Z"), new Date("2019-04-27T10:20:30Z"), new Date("2019-04-28T10:20:30Z")];

describe("<Navigation />", () => {
  it("should display component", () => {
    const wrappedComponent = shallow(<Navigation weekDates={mockWeekDates} />);

    expect(wrappedComponent).toMatchSnapshot();
  });
});
