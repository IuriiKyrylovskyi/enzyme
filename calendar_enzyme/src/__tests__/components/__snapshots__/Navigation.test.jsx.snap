// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> should display component 1`] = `
<header
  className="calendar__header"
>
  <div
    className="calendar__day-label day-label"
    key="25.308976727560896"
  >
    <span
      className="day-label__day-name"
    >
      Mon
    </span>
    <span
      className="day-label__day-number"
    >
      22
    </span>
  </div>
  <div
    className="calendar__day-label day-label"
    key="58.34905949030602"
  >
    <span
      className="day-label__day-name"
    >
      Tue
    </span>
    <span
      className="day-label__day-number"
    >
      23
    </span>
  </div>
  <div
    className="calendar__day-label day-label"
    key="13.645600875943153"
  >
    <span
      className="day-label__day-name"
    >
      Wed
    </span>
    <span
      className="day-label__day-number"
    >
      24
    </span>
  </div>
  <div
    className="calendar__day-label day-label"
    key="92.38295979760606"
  >
    <span
      className="day-label__day-name"
    >
      Thu
    </span>
    <span
      className="day-label__day-number"
    >
      25
    </span>
  </div>
  <div
    className="calendar__day-label day-label"
    key="93.05874259812445"
  >
    <span
      className="day-label__day-name"
    >
      Fri
    </span>
    <span
      className="day-label__day-number"
    >
      26
    </span>
  </div>
  <div
    className="calendar__day-label day-label"
    key="94.00819837691228"
  >
    <span
      className="day-label__day-name"
    >
      Sat
    </span>
    <span
      className="day-label__day-number"
    >
      27
    </span>
  </div>
  <div
    className="calendar__day-label day-label"
    key="85.83092512681958"
  >
    <span
      className="day-label__day-name"
    >
      Sun
    </span>
    <span
      className="day-label__day-number"
    >
      28
    </span>
  </div>
</header>
`;
